lib/webpack/create-chain.js

```diff
@@ -16,6 +16,7 @@ module.exports = function (cfg, configName) {
     fileHash = needsHash ? '.[hash:8]' : '',
     chunkHash = needsHash ? '.[contenthash:8]' : '',
     resolveModules = [
+      'node_modules',
```

Comment:
Hi, isn't this same as "appPaths.resolve.app('node_modules')" ?

Reply:
https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-service/lib/config/base.js#L43-L46
Vue CLI also has the same config :)